src/components/iap/stripe-legacy-iap-prices.json

```diff
@@ -0,0 +1,14 @@
+{
+  "collectionName": "components_iap_stripe_legacy_iap_prices",
+  "info": {
+    "displayName": "stripeLegacyIapPrices",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "stripeLegacyIapPrices": {
+      "type": "string",
```

Comment:
Missing `minLength` for validation?

Reply:
Added a regex checking that the strings start with `price_` or `plan_` instead of having a minimum length.